app/src/main/res/layout/item_detail_bill.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:car_view="http://schemas.android.com/tools"
+    >
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.framgia.fsalon.screen.billdetail.BillDetailViewModel"
+            />
+
+        <variable
+            name="item"
+            type="com.framgia.fsalon.data.model.BillItemResponse"/>
+
+        <variable
+            name="position"
+            type="int"/>
+
+    </data>
+
+    <android.support.v7.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/dp_8"
+        car_view:cardElevation="@dimen/dp1"
+        >
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center"
+            android:orientation="horizontal"
+            >
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:background="@color/colorPrimary"
+                android:gravity="center"
+                android:minWidth="@dimen/dp_20"
+                android:padding="@dimen/dp_2"
+                android:text="@{position + 1 + ``}"
+                android:textColor="@color/color_white"
+                android:textSize="@dimen/sp_20"/>
+
+            <LinearLayout
+                android:layout_width="0dp"
+                android:layout_height="match_parent"
+                android:layout_weight="1"
+                android:orientation="vertical"
+                android:paddingLeft="@dimen/dp_3"
+                android:paddingStart="@dimen/dp_3">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="0dp"
+                    android:layout_margin="@dimen/dp_1"
+                    android:layout_marginEnd="@dimen/dp_30"
+                    android:layout_marginRight="@dimen/dp_30"
+                    android:layout_weight="1"
```

Comment:
WARNING: [NestedWeights] Nested weights are bad for performance
 Layout weights require a widget to be measured twice. When a LinearLayout with non-zero weights is nested inside another LinearLayout with non-zero weights, then the number of measurements increase exponentially.